twitter/models/twitter_post.go

```diff
@@ -73,18 +73,18 @@ type TwitterPost struct {
 	Place    string
 
 	// content
-	Cashtags    pq.StringArray `gorm:"type:varchar(64)[]"`
-	Hashtags    pq.StringArray `gorm:"type:varchar(64)[]"`
+	Cashtags    pq.StringArray `gorm:"type:text[]"`
+	Hashtags    pq.StringArray `gorm:"type:text[]"`
```

Comment:
Why have you changed all field types to text? This will also require more disk space and could perhaps be avoided for hashtags and cashtags

Reply:
Do you know if there is a hashtags size limit from twitter? Else it would just be a source of error for the future to limit it.
Imo